### ISSUE TITLE
Error on scan device command

### ISSUE THREAD

**Issue by rasmusbe** [mdeweerd/zha-toolkit#57]
```yaml
service: zha_toolkit.scan_device
data:
  ieee: light.livingroom_window_right
  endpoint: 1
```

gives:
```
This error originated from a custom integration.

Logger: homeassistant.helpers.script.websocket_api_script
Source: custom_components/zha_toolkit/scan_device.py:91
Integration: 🧰 ZHA Toolkit Service (documentation, issues)
First occurred: 13:07:26 (5 occurrences)
Last logged: 13:07:52

websocket_api script: Error executing script. Unexpected error for call_service at pos 1: local variable 'endpoint' referenced before assignment
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 379, in _async_step
    await getattr(self, handler)()
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 582, in _async_call_service_step
    await service_task
  File "/usr/src/homeassistant/homeassistant/core.py", line 1634, in async_call
    task.result()
  File "/usr/src/homeassistant/homeassistant/core.py", line 1671, in _execute_service
    await cast(Callable[[ServiceCall], Awaitable[None]], handler.job.target)(
  File "/config/custom_components/zha_toolkit/__init__.py", line 692, in toolkit_service
    raise handler_exception
  File "/config/custom_components/zha_toolkit/__init__.py", line 656, in toolkit_service
    await handler(
  File "/config/custom_components/zha_toolkit/__init__.py", line 741, in command_handler_default
    await default.default(
  File "/config/custom_components/zha_toolkit/default.py", line 33, in default
    await handler(app, listener, ieee, cmd, data, service, params, event_data)
  File "/config/custom_components/zha_toolkit/scan_device.py", line 394, in scan_device
    scan = await scan_results(device, endpoints, manufacturer=manf)
  File "/config/custom_components/zha_toolkit/scan_device.py", line 91, in scan_results
    ep_result.append(endpoint)
UnboundLocalError: local variable 'endpoint' referenced before assignment
```

I'm using a Silicon Labs EZSP stick and trying to scan a Signify (Philips Hue) E27 light (LCT015)

**Comment by mdeweerd** [comment]
Should be fixed in v0.8.4 .

**Comment by rasmusbe** [comment]
Seems to be working in the new version 👍
